packages/cli/register/src/ir-to-fdr-converter/convertPackage.ts

```diff
@@ -679,7 +678,10 @@ function convertHttpEndpointExample({
             return undefined;
         })
         .filter(isNonNullish);
-    const example = userSpecifiedExample?.example ?? autogeneratedExample.example;
+    const example = userSpecifiedExample?.example ?? autogeneratedExample?.example;
+    if (example === undefined) {
+        throw new Error("Example is unexpectedly undefined.");
```

Comment:
we should not be throwing in this case and blocking docs generation -- can we return undefined and handle that upstream or rearchitect the code so that we don't have to throw?

Reply:
Checks out. Fixed.